examples/fromjson/main.go

```diff
@@ -9,6 +9,22 @@ import (
 	"github.com/grafadruid/go-druid/builder"
 )
 
+func loadAndExecute(d *druid.Client, qry []byte) {
+	var err error
+	var q builder.Query
+	var results interface{}
+	q, err = d.Query().Load(qry)
+	if err != nil {
+		log.Fatalf("Load failed, %s (query: %s)", err, string(qry))
+	}
+	spew.Dump(q)
+	_, err = d.Query().Execute(q, &results)
+	if err != nil {
+		log.Fatalf("Load failed, %s (query: %s)", err, string(qry))
```

Comment:
maybe great to distinguish that case with a message like `"Execute failed", %s (query : %s)`

Reply:
which case are you talking about? I didn't get the context.

Reply:
nm.. i got it now, i will fix the comment 😄